lib/ansible/modules/cloud/misc/qemu_img.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2013, Jeroen Hoekx <jeroen.hoekx@dsquare.be> - Toshaan Bharvani <toshaan@vantosh.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+DOCUMENTATION = '''
+---
+author: Jeroen Hoekx & Toshaan Bharvani
+module: qemu_img
+short_description: Create qemu images
+description:
+  - "This module creates images for qemu."
+version_added: "2.2"
+options:
+  dest:
+    description: The image file to create or remove
+    required: true
+  format:
+    description: The image format
+    default: qcow2
+    required: false
+  options:
+    description: The disk format options
+    required: false
+  size:
+    description: The size of the image
+    required: false
+  state:
+    choices: [ "absent", "present", "resize" ]
+    description:
+    - If the image should be present or absent
+    default: present
+    required: false
+notes:
+  - This module does not change the type of the image.
+'''
+
+EXAMPLES = '''
+  - description: Create a raw image of 5M.
+    code: qemu_img dest=/tmp/testimg size=5 format=raw
+
+  - description: Enlarge the image to 6M.
+    code: qemu_img dest=/tmp/testimg size=6 format=raw
+
+  - description: Remove the image
+    code: qemu_img dest=/tmp/testimg state=absent
```

Comment:
Probably best to use the YAML syntax for examples here.

You no longer need to use `description:` and `code`.
You can just add your tasks as in a normal playbook with `name:` and your module action.